docs/sphinx/source/whatsnew/v0.11.2.rst

```diff
@@ -20,7 +24,10 @@ Bug fixes
 Documentation
 ~~~~~~~~~~~~~
 * Edited docstrings for :py:func:`~pvlib.pvsystem.dc_ohms_from_percent` and
-  :py:func:`~pvlib.pvsystem.dc_ohmic_losses` for clarity. (:issue:`1601`, :pull:`2229`)
+  :py:func:`~pvlib.pvsystem.dc_ohmic_loss` for clarity. (:issue:`1601`, :pull:`2229`)
+* Clarified that :py:meth:`~pvlib.pvsystem.PVSystem.get_irradiance` and
+  :py:meth:`~pvlib.pvsystem.Array.get_irradiance` do not accept tuple of float
+  as input. (:pull:`2227`)
```

Comment:
Per the above discussion, I think this statement is not currently true, nor should it be a goal.  Tuple of float is the form corresponding to multiple arrays and a single timestamp of array-specific operating conditions:

```python
In [15]: arrays = [
    ...:     pvlib.pvsystem.Array(pvlib.pvsystem.FixedMount(surface_tilt=10)),
    ...:     pvlib.pvsystem.Array(pvlib.pvsystem.FixedMount(surface_tilt=20)),
    ...: ]
    ...: system = pvlib.pvsystem.PVSystem(arrays=arrays)
    ...: 
    ...: dni = (1000, 200)  # sunny on array 1, cloudy on array 2
    ...: dhi = (200, 300)
    ...: ghi = (800, 400)
    ...: zenith = 50
    ...: azimuth = 170
    ...: 
    ...: system.get_irradiance(zenith, azimuth, dni, ghi, dhi)
Out[15]: 
(OrderedDict([('poa_global', 992.7295708775583),
              ('poa_direct', 764.0235366741815),
              ('poa_diffuse', 228.7060342033768),
              ('poa_sky_diffuse', 227.1868095045976),
              ('poa_ground_diffuse', 1.519224698779198)]),
 OrderedDict([('poa_global', 482.67340206708127),
              ('poa_direct', 172.40899902290855),
              ('poa_diffuse', 310.26440304417275),
              ('poa_sky_diffuse', 307.2490340834682),
              ('poa_ground_diffuse', 3.0153689607045786)]))

```